examples/12_external_data/pet_drifter_loopers.py

```diff
@@ -0,0 +1,153 @@
+"""
+Colocate looper with eddy from altimetry
+========================================
+
+All loopers data used in this example are a subset from the dataset describe in this article
+[Lumpkin, R. : Global characteristics of coherent vortices from surface drifter trajectories](https://doi.org/10.1002/2015JC011435)
+"""
+
+import re
+
+import numpy as np
+import py_eddy_tracker_sample
+from matplotlib import pyplot as plt
+from matplotlib.animation import FuncAnimation
+
+from py_eddy_tracker import data
+from py_eddy_tracker.appli.gui import Anim
+from py_eddy_tracker.observations.tracking import TrackEddiesObservations
+
+
+# %%
+class VideoAnimation(FuncAnimation):
+    def _repr_html_(self, *args, **kwargs):
+        """To get video in html and have a player"""
+        content = self.to_html5_video()
+        return re.sub(
+            r'width="[0-9]*"\sheight="[0-9]*"', 'width="100%" height="100%"', content
+        )
+
+    def save(self, *args, **kwargs):
+        if args[0].endswith("gif"):
+            # In this case gif is use to create thumbnail which are not use but consume same time than video
```

Comment:
gif is used to create a thumbnail which is not used but consume same time as video